packages/docusaurus/src/server/__tests__/__snapshots__/brokenLinks.test.ts.snap

```diff
@@ -20,13 +20,13 @@ Exhaustive list of all broken links found:
 exports[`brokenLinks getBrokenLinksErrorMessage with potential layout broken links 1`] = `
 "Docusaurus found broken links!
 
-Please check the pages of your site in the list bellow, and make sure you don't reference any path that does not exist.
+Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
 Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
 
 It looks like some of the broken links we found appear in many pages of your site.
 Maybe those broken links appear on all pages through your site layout?
 We recommend that you check your theme configuration for such links (particularly, theme navbar and footer).
-Frequent broken links are linking to: 
```

Comment:
Since you removed  whitespace in this snapshot, you also need to remove it in the code itself, right [here](https://github.com/facebook/docusaurus/blob/c7c90a4ffb2d4593ac05dd694540032d523fa07b/packages/docusaurus/src/server/brokenLinks.ts#L133). Then the build will succeed again.